source/kraken/fill_disruption_from_chaos.cpp

```diff
@@ -357,26 +381,30 @@ struct functor_add_vj {
             //this vehicle journey won't ever circulate
             return false;
         }
-        vj->adapted_validity_pattern = pt_data.get_or_create_validity_pattern(tmp_vp);
+        auto meta_vj_it = pt_data.meta_vj.find(vj_ref.uri);
+        if (meta_vj_it != pt_data.meta_vj.cend()) {
+            push_back_unique(pt_data.meta_vj[vj_ref.uri]->adapted_vj, &vj);
```

Comment:
adapted_vj should be always unique in vj_ref's adapted_vj list... maybe push_back_unique is useless here